### PR TITLE
ADDED:BGT data stories

### PR DESCRIPTION
**Pull request by kad-rowla**
Only added some tiles on the BGT labs page but maybe some Dutch to be reviewed on the data.labs stories themselves.

### CODE REVIEW

File: cases/bgt-ld.md [hunk 1]
```diff
@@ -54,6 +54,22 @@ Echter, om meer inzicht te bieden in de mogelijkheden hebben we voorbeelden gema
       <div class="card-title">BGT Wegwerkzaamheden</div>
       <div class="card-description">Data Story die laat zien hoe de BGT de planning van wegwerkzaamheden kan ondersteunen.</div>
     </div>
+  </a>
+  <a href="https://data.labs.kadaster.nl/kadaster/-/stories/hoogspanningsmast-nl" >
+    <div class="card">
+      <div class="card-type">Story</div>
+      <img class="card-image" src="/assets/images/hoogspanning.jpg" alt="Hoogspanningsmasten">
+      <div class="card-title">BGT Hoogspanningsmasten Kaart</div>
+      <div class="card-description">Dit data story toont het totaal aantal hoogspanningsmasten in Nederland en visualiseert deze op een kaart.</div>
```
Comment (kad-wenner) [inline cases/bgt-ld.md:63]: Deze data story

File: cases/bgt-ld.md [hunk 2]
```diff
@@ -54,6 +54,22 @@ Echter, om meer inzicht te bieden in de mogelijkheden hebben we voorbeelden gema
...
+    </div>
+  </a>
+  <a href="https://data.labs.kadaster.nl/kadaster/-/stories/bgt-plustype-bomen-gemeenten" >
+    <div class="card">
+      <div class="card-type">Story</div>
+      <img class="card-image" src="/assets/images/bomen.jpg" alt="BomenPlusData">
+      <div class="card-title">BGT Bomen als Plus-Type Data</div>
+      <div class="card-description">Data story dat laat zien welke gemeenten boom plus-type data in heel Nederland vastleggen.</div>
```
Comment (kad-wenner) [inline cases/bgt-ld.md:71]: Deze data story laat zien welke gemeenten data over BGT plus-type bomen in heel Nederland vastleggen

File: data/bag-ld/examples/pand_instantie_wide.trig [hunk 3]
```diff
@@ -161,7 +161,7 @@ graph:instanties {
         geo:asWKT "<http://www.opengis.net/def/crs/EPSG/0/28992> Point((206486.218 530180.399 0.0))"^^geo:wktLiteral ];
     bag:status status:verblifsobjectInGebruik;
     bag:gebruikersdoel "woonfunctie";
-    bag:gebruikersdoel "industriefunctie"; #there are two of these for each registration. Not sure of the syntax maybe - bag:industrieFunctie?
+    bag:gebruikersdoel "industriefunctie"; #@Wouter there are two of these for each registration. Not sure of the syntax maybe - bag:industrieFunctie?
```
Comment (kad-wenner) [inline data/bag-ld/examples/pand_instantie_wide.trig:164]: Wrong branch?

File: data/bag-ld/temporal/Temporal.xml [hunk 4]
```diff
@@ -644,4 +644,121 @@
             <Objecten:documentnummer>228124</Objecten:documentnummer>
         </Objecten:Standplaats>
     </sl-bag-extract:bagObject>
+</sl:stand>
```
Comment (kad-wenner) [inline data/bag-ld/temporal/Temporal.xml:647]: Wrong branch?

File: data/bag-ld/examples/pand_instantie_wide.trig (hunk 3 above)
Comment (kad-rowla) [reply data/bag-ld/examples/pand_instantie_wide.trig:164]: Yes, but have made changes on other repository too.

File: data/bag-ld/temporal/Temporal.xml (hunk 4 above)
Comment (kad-rowla) [reply data/bag-ld/temporal/Temporal.xml:647]: Yes, but have made changes on other repository too.

**Pull request merged**